domain/infosync/info.go

```diff
@@ -183,15 +183,9 @@ func GlobalInfoSyncerInit(ctx context.Context, id string, serverIDGetter func()
 	if err != nil {
 		return nil, err
 	}
-	if etcdCli != nil {
-		is.labelRuleManager = initLabelRuleManager(etcdCli.Endpoints())
```

Comment:
The label manager also has this problem in v5.3.0.

Reply:
Added the label